frontend/src/components/object/ObjectMetadata.vue

```diff
@@ -3,31 +3,75 @@ import { storeToRefs } from 'pinia';
 import { onMounted, ref, watch } from 'vue';
 
 import GridRow from '@/components/form/GridRow.vue';
-import { useMetadataStore, useVersionStore } from '@/store';
+import ObjectMetadataTagForm from '@/components/object/ObjectMetadataTagForm.vue';
```

Comment:
We have an index.ts directory repackager in /components/object - we should route components through there instead of directly importing the component.